Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "graft"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
Big jump :-p, make it 0.2.0 or something.

Reply:
Done